plugin/backend/ipvariant.d

```diff
@@ -730,20 +723,26 @@ import std.container.array;
                 with (dtor_body(impl_nsname ~"_Impl")) { //Generate destructor
              	    sep;
                 }
+
+                with(func_body("void", "Regenerate")) {
+                    stmt(E("fum.V0") = E("randomGenerator->generate(\"Requirer fum V0\")"));
```

Comment:
tip: it is possible to create strings using back ticks for strings, `
Which is useful in circumstances like this.